zsh-bash-completions-fallback.plugin.zsh

```diff
@@ -1,6 +1,17 @@
 [[ -o interactive ]] || return 0
 
 _bash_completions_getter_path=${0:A:h}/bash-completions-getter.sh
+_bash_completions_dirs=(
+    ${(@s/:/)${XDG_DATA_DIRS}} # in NixOS, it contains /run/current-system/sw/share
+    ~/.local/share
```

Comment:
`~/.local` stuff should probably have more priority than `$XDG_DATA_DIRS` too.

Reply:
I agree.